src/containers/AdminCampaignStats/CampaignStat.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card, CardContent, CardHeader } from "@material-ui/core";
+import { css, StyleSheet } from "aphrodite";
+import React from "react";
+
+import LoadingIndicator from "../../components/LoadingIndicator";
+
+const inlineStyles = StyleSheet.create({
+  cardContent: {
+    textAlign: "center",
+    height: "50%",
+    padding: "5px"
+  },
+  colorRed: { color: "red" }
+});
+
+export interface CampaignStatProps {
+  title: string;
+  loading?: boolean;
+  error?: string;
+  count?: any;
+  highlight?: boolean;
+}
+
+export const CampaignStat: React.FC<CampaignStatProps> = (props) => {
+  // shrink text if large top line number
+  const maxLargeNumberLength = 4;
+  const count = props.count && props.count.toString();
+  const heading =
+    count && !count.includes("%") && count.length > maxLargeNumberLength
```

Comment:
This component gets used for both the top line stats and the deliverability stats below. The deliverability ones always include a percent with the raw count. Since the deliverability ones have plenty of space for long strings, can skip resizing those